src/procedures/SignDisableControllerAck.ts

```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignDisableControllerAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for disabling the controller of a security token
```

Comment:
Procedure that signs an acknowledgement to permanently disable a Security Token's Controller functionality